### PR TITLE
Create 2020-06-07-pork-larb.md

### PR DESCRIPTION
**Pull request by zmagg**

### CODE REVIEW

File: _posts/meat/2020-06-07-pork-larb.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: Pork Larb
+category: meat
+tags: recipeish
+---
+
+Writing this up so that Buro can make it too. <3
+
+Ingredients
+
+* 1 pound ground pork
```
Comment (mookerji) [inline _posts/meat/2020-06-07-pork-larb.md:12]: These ingredients can be grouped a bit. Like: 

```
For serving: 
* 1/2 cup - 1 cup white jasmine/sticky rice, or
* lettuce / chard leaves for serving (optional)

Garnishes (optional): 
* 1/2 bunch green onions, chopped (optional)
* 1/2 bunch cilantro, chopped (optional)
* few sprigs mint, chopped (optional) 

```

File: _posts/meat/2020-06-07-pork-larb.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* lettuce / chard leaves for serving (optional)
+
+Tools
+
+* 1 wok / cast iron pan
+* 1 blade grinder
+
+Technique
+
+* In dry pan, toast rice until all grains are evenly light brown. Stir frequently over medium heat.
```
Comment (mookerji) [inline _posts/meat/2020-06-07-pork-larb.md:33]: Would it be possible to group or consolidate these instructions? For example, the prep steps as a separated set of bullet points. It'll make it easier to follow, especially 'on mobile'.

File: _posts/meat/2020-06-07-pork-larb.md (hunk 1 above)
Comment (zmagg) [reply _posts/meat/2020-06-07-pork-larb.md:12]: the rice isn't for serving tho it's for toasting

**Comment by mookerji** [comment]
Derp. You’re right!

File: _posts/meat/2020-06-07-pork-larb.md (hunk 1 above)
Comment (mookerji) [reply _posts/meat/2020-06-07-pork-larb.md:12]: Oh derp

**Pull request merged**